cmd/kubelet/app/options/options.go

```diff
@@ -385,7 +388,8 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.StringVar(&f.VolumePluginDir, "volume-plugin-dir", f.VolumePluginDir, "The full path of the directory in which to search for additional third party volume plugins")
 	fs.StringVar(&f.LockFilePath, "lock-file", f.LockFilePath, "<Warning: Alpha feature> The path to file for kubelet to use as a lock file.")
 	fs.BoolVar(&f.ExitOnLockContention, "exit-on-lock-contention", f.ExitOnLockContention, "Whether kubelet should exit upon lock-file contention.")
-	fs.StringVar(&f.SeccompProfileRoot, "seccomp-profile-root", f.SeccompProfileRoot, "<Warning: Alpha feature> Directory path for seccomp profiles.")
+	fs.StringVar(&f.SeccompProfileRoot, "seccomp-profile-root", f.SeccompProfileRoot, "[Experimental] Directory path for seccomp profiles.")
+	fs.StringVar(&f.DefaultSeccompProfile, "default-seccomp-profile", f.DefaultSeccompProfile, "[Experimental] The default seccomp profile")
```

Comment:
If it's experimental, should it be behind a feature flag? I'm not sure this needs to be considered experimental...